src/runtime_src/core/tools/common/ReportDebugIpStatus.cpp

```diff
@@ -76,6 +76,9 @@ static const char* debugIpNames[DEBUG_IP_TYPE_MAX] = {
   "Accelerator Deadlock Detector (accel_deadlock_detector)"
 };
 
+static size_t cuNameMaxStrLen[DEBUG_IP_TYPE_MAX] = {0};
```

Comment:
The report functions for performance monitors use "cuNameMaxStrLen" to format report output. Those functions now use boost::property_tree passed to "writeReport" and so  don't need to be member method of "DebugIpStatusCollector" class. So, I made cuNameMaxStrLen a global so that it can be usd by report methods.